chart/values.yaml

```diff
@@ -0,0 +1,18 @@
+domain: "###ZARF_VAR_DOMAIN###"
+
+# customNetworkPolicies:
```

Comment:
Are these commented on purpose? suggest removing so there isn't confusion with what UDS package CR would provide.

Reply:
These are commented on purpose, we're doing this across all packages as an example of how to use the custom net pols value